backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -28,6 +28,16 @@ class BibTexCommonSchema(Schema):
     booktitle = fields.Method("get_book_title")
     collaboration = fields.Method("get_collaboration")
     doc_type = fields.Raw()
+    document_type_map = {
+        "article": "article",
+        "book": "book",
+        "book chapter": "inbook",
+        "proceedings": "proceedings",
+        "report": "techreport",
+        "note": "article",
+        "conference paper": lambda data: "article" if get_value(
+            data, "publication_info.journal_title") else "inproceedings"
```

Comment:
I guess it wasn't clear, but I suggested to do this because the same pattern is also used for `thesis` in `get_document_type`. You should also put this further down in the class, as it's not part of the serialized fields.